settings.py

```diff
@@ -8,9 +8,10 @@ def load(module="settings_default"):
     global settings
     try:
         m = __import__(module, "settings")
-        settings = m.settings
+    except ImportError:
+        raise SystemExit('Error! No file with configurations found: settings_default.py')
     except Exception:
-        settings = {}
+        raise SystemExit('Error! File found, but error in settings. Check settings_sample.py file')
```

Comment:
```suggestion
        raise SystemExit('Error! File found, but error in settings_default.py. Check settings_sample.py file for examples')
```
Crec que queda més clar on és l'error si poses el nom del fitxer sencer, com a sobre.